postgres/tests/compose/resources/02_setup.sh

```diff
@@ -43,12 +43,23 @@ psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" "$DBNAME" <<-'EOSQL'
     RETURNS NULL ON NULL INPUT
     SECURITY DEFINER;
 
+    CREATE OR REPLACE FUNCTION datadog.explain_statement_noaccess(l_query text, out explain JSON) RETURNS SETOF JSON AS
```

Comment:
Should add a comment to this one just clarifying it's not part of the actual setup.

Reply:
Good point, added